### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by shaikhhamza11**
1)Added card components

1)Card with text 
2)Image card 
3)Text overlays
4)Horizontal card
5) card with Dismis
Screenshot (26)](https://user-images.githubusercontent.com/51785892/152991870-7a3d3552-f391-47ee-b087-6253627ceeeb.png)
es.githubusercontent.com/51785892/152991845-12a7ea86-f042-4b96-855e-b3b149461234.png)
Screenshot (27)

Screenshot (25)
s

### CODE REVIEW

File: components/alert.css [hunk 1]
```diff
@@ -1,8 +1,9 @@
 .alert-container {
   width: 90%;
+  border: 2px solid var(--color-primary);
 }
 .alert {
-  width: 100%;
+  width: 90%;
   padding: 1.5rem;
   position: relative;
   border-left: 0.8rem solid #bababa;
```
Comment (praveenreddy1798) [inline components/alert.css]: can we use color variable here instead of using it directly ?

File: components/cards.css [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  padding: 2rem;
+  border-radius: 1rem;
+  min-height: 22rem;
+  margin: 1rem;
+}
+
+.card-text h3 {
+  font-size: 2rem;
+  font-weight: bold;
+  border-bottom: 2px solid black;
```
Comment (praveenreddy1798) [inline components/cards.css]: can we use color variable here ?

File: components/cards.css [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+}
+.card-container .card-text p,
+.card-container .image-card p {
+  font-size: 1.8rem;
+}
+
+.card {
+  min-width: 22rem;
+  margin: 2rem auto;
+  border: 3px solid rgba(0, 0, 0, 0.125);
```
Comment (praveenreddy1798) [inline components/cards.css]: color variable

**Comment by praveenreddy1798** [comment]
You have used color variables directly at many places, mostly for shadows and borders. Is there any specific reason to it so that even i can get some insights or is it just because those colors are being used only there and no where else ?

Another suggestion is can we create flexbox classes in utils and use them instead of defining individually across component containers. Maybe you can consider for your upcoming components so that it can reduce repeated code.

Just these two things, otherwise looks great

**Comment by shaikhhamza11** [comment]
> You have used color variables directly at many places, mostly for shadows and borders. Is there any specific reason to it so that even i can get some insights or is it just because those colors are being used only there and no where else ?
> 
> Another suggestion is can we create flexbox classes in utils and use them instead of defining individually across component containers. Maybe you can consider for your upcoming components so that it can reduce repeated code.
> 
> Just these two things, otherwise looks great

Yes because these color are being used only there so no need to create  variable. Also I have created flex utilities classes but in some components I don't use them as I want more control of the components in media queries .

**Review by praveenreddy1798 — APPROVED** [review]

**Review by tanishk15sharma — APPROVED** [review]
good to merge

**Pull request merged**